### PR TITLE
Feature/touchups utils

### PR DESCRIPTION
**Pull request by mosheman5**
- Create metadata_processing.py file with few utils I found helpful when creating a dataset and I believe can be used in other usecases
- break down the mega utils.py file into smaller files, each utility wrt to its topic
- fix tests and references in conf files

### CODE REVIEW

File: tests/test_data.py [hunk 1]
```diff
@@ -1,7 +1,4 @@
-import sys
-sys.path.append('../soundbay')
 from hydra.experimental import compose, initialize
-import pytest
 from random import randint
```
Comment (Z30G0D) [inline tests/test_data.py:2]: why remove pytest??

**Review by Z30G0D — CHANGES_REQUESTED** [review]
Few remarks,
Did we recently check our test? :E

File: soundbay/utils/metadata_processing.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        dfTemp['filename'] = re.search(
+            "\.Table.1.selections", file.as_posix()).group()
+        dfTemp['FirstCallTime'] = np.amin(dfTemp['Begin Time (s)'])
+        dfTemp['LastCallTime'] = np.amax(dfTemp['End Time (s)'])
+        metadata.append(dfTemp)
+
+    metadata = pd.concat(metadata)
+    metadata.rename(columns={'Begin Time (s)': 'begin_time', 'End Time (s)': 'end_time'}, inplace=True)
+    print('Number of Labels:', metadata.shape[0])
+    for tag in positive_tag_names:
```
Comment (noamzbr) [inline soundbay/utils/metadata_processing.py:35]: what does this loop do?

File: soundbay/utils/metadata_processing.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        return ['begin_time', 'end_time', 'filename', 'call_length', 'label']
+
+    orig_cols = df.columns.tolist()
+    default_cols = get_metadata_fields()
+    remaining = list(set(orig_cols) - set(default_cols))
+    new_cols = default_cols + remaining
+    return df[new_cols]
+
+
+def correct_call_times_wrt_duration(df: pd.DataFrame, audio_files_path: str):
```
Comment (noamzbr) [inline soundbay/utils/metadata_processing.py:139]:
```suggestion
def correct_call_times_with_duration(df: pd.DataFrame, audio_files_path: str):
```
?

File: tests/test_data.py (hunk 1 above)
Comment (mosheman5) [reply tests/test_data.py:2]: no usage in the file

**Comment by mosheman5** [comment]
> 

we did, they work. you have to install the package using `pip install . -e` in order to run them properly

File: soundbay/utils/metadata_processing.py (hunk 2 above)
Comment (mosheman5) [reply soundbay/utils/metadata_processing.py:35]: no idea, `raven_to_df_annotations` is an old leftover from utils. 
Perhaps we should delete it and create the moz dataset using the smaller new util functions.

File: soundbay/trainers.py [hunk 4]
```diff
@@ -1,9 +1,10 @@
 from typing import Union, Generator, Tuple
 import torch
 import torch.utils.data
-from soundbay.utils import Logger, app
+from soundbay.utils.logging import Logger
```
[the remark this answers is not in the document]
Comment (mosheman5) [reply soundbay/trainers.py]: done :)

**Review by Z30G0D — APPROVED** [review]

**Pull request merged**